rest/api.go

```diff
@@ -27,9 +27,19 @@ import (
 	"github.com/gorilla/mux"
 )
 
+type HandleMethod string
+
 const (
 	defaultLogPrefix     = "rest "
 	defaultDocsDirectory = "_docs/"
+
+	// Handler names
+	HandleCreate     HandleMethod = "create"
+	HandleRead       HandleMethod = "read"
+	HandleUpdate     HandleMethod = "update"
+	HandleDelete     HandleMethod = "delete"
+	HandleReadList   HandleMethod = "readList"
+	HandleUpdateList HandleMethod = "updateList"
```

Comment:
This is a nit, but you just need to put `HandleMethod` on the first one and it will be implied for the rest.

```go
	HandleCreate     HandleMethod = "create"
	HandleRead                    = "read"
	HandleUpdate                  = "update"
	HandleDelete                  = "delete"
	HandleReadList                = "readList"
	HandleUpdateList              = "updateList"
```